t/destructive/testers.go

```diff
@@ -0,0 +1,33 @@
+package destructive
+
+import (
+	"github.com/vishvananda/netlink"
+)
+
+// NetworkConnectionTester removes and restores network connection.
+type NetworkConnectionTester struct {
```

Comment:
I'd call it `NetworkController` or something like that.
It doesn't test anything, does it?

Reply:
it doesnt) but i also not sure about Controller name

Reply:
You can just call it `NetworkLink` and have methods `PutUp` and `PutDown`, if that's the idea.

Reply:
the idea was to have something abstract. another "tester" could implement increased latency or random packets drop

Reply:
Apple has the exact tool for that named ”Network Link Conditioner”
:)

Reply:
i named it NetworkConnectionController